### PR TITLE
[DUOS-2804] Improve dataset statistics page

### PR DESCRIPTION
**Pull request by fboulnois**
### Addresses

https://broadworkbench.atlassian.net/browse/DUOS-2804

### Summary

Improve dataset statistics page:
* ensure number of participants is listed
* description is pulled from study if not on dataset
* hide data custodians if none are available
* hide pi if none are available

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/DatasetStatistics.jsx [hunk 1]
```diff
@@ -78,52 +78,28 @@ export default function DatasetStatistics(props) {
...
                 <div style={Styles.SMALL_BOLD}>Principal Investigator: </div>
                 <div style={Styles.SMALL_BOLD}>
-                  {dataset?.study?.piName || get(
-                    find(dataset?.properties, (p) => {
-                      return p.propertyName === 'Principal Investigator(PI)';
-                    }),
-                    'propertyValue',
-                    ''
-                  )}
+                  {extract('Principal Investigator(PI)') || dataset?.study?.piName}
```
Comment (rushtong) [inline src/pages/DatasetStatistics.jsx:94]: One of the things I realized in this code is that the `dataset` object isn't a real dataset as we understand it, but instead is a `DatasetDTO` [object](https://github.com/DataBiosphere/consent/blob/develop/src/main/java/org/broadinstitute/consent/http/models/DatasetMetrics.java#L8) which is a deprecated class. We'll definitely need some back end work to make this more accurate.

**Review by rushtong — APPROVED** [review]
Looks good 👍🏽

**Review by PintoGideon — APPROVED** [review]
Tested and everything works 👍🏽

File: src/pages/DatasetStatistics.jsx [hunk 2]
```diff
@@ -78,52 +78,28 @@ export default function DatasetStatistics(props) {
...
               {LINE}
               <div style={{ fontSize: Theme.font.size.small, padding: '1rem' }}>
-                {get(
-                  find(dataset?.properties, (p) => {
-                    return p.propertyName === 'Description';
-                  }),
-                  'propertyValue',
-                  ''
-                )}
+                {extract('Dataset Description') || dataset?.description || dataset?.study?.description }
```
Comment (PintoGideon) [inline src/pages/DatasetStatistics.jsx]: Should we add a default value, such as "N/A", for situations where no values are provided, or should we gray out the properties? Just a thought. No need to implement it in this PR, of course.

File: src/pages/DatasetStatistics.jsx (hunk 1 above)
Comment (fboulnois) [reply src/pages/DatasetStatistics.jsx:94]: Yes, this is why I extract the dataset directly instead using `DataSet.getDataSetsByDatasetId`

**Review by aarohinadkarni — APPROVED** [review]
looks good 👍

**Pull request merged**